src/components/input/input.scss

```diff
@@ -34,7 +34,9 @@ $md-input-underline-disabled-background-image: linear-gradient(to right,
   visibility: visible;
   padding-bottom: 5px;
   transform: translateY(-100%) scale(0.75);
-
+  transition: width 150ms ease-in-out;
```

Comment:
So this will overwrite or be overwritten by the transition already on the `.md-input-placeholder`. The transition should be added to line 125 AND use material transition functions.